Modules/Server/Sources/PocketCastsServer/Public/AppSettings.swift

```diff
@@ -0,0 +1,16 @@
+import PocketCastsUtils
+
+/// Model type for synced & stored App Settings
+public struct AppSettings: JSONCodable {
+    @ModifiedDate public var openLinks: Bool
+
+    @ModifiedDate public var rowAction: PrimaryRowAction
+
+    static var defaults: AppSettings {
+        return AppSettings(openLinks: true, rowAction: .download)
```

Comment:
Yeah, I was planning to add this in a follow up PR since this is just the base storage change. I believe those changes will be completely additive so the core design here will remain the same. Do you think it would be better to include this now so settings aren't lost if someone turns the flag on?

Reply:
If you're curious about what this might look like, I pushed a commit here: https://github.com/Automattic/pocket-casts-ios/commit/36951d8957e86539b76250f480944e35a3994988

I think I'd like to submit that separately, though, and make sure I can include a modification to default sync times for those settings as described here: https://github.com/Automattic/pocket-casts-android/pull/1798

Reply:
> Do you think it would be better to include this now so settings aren't lost if someone turns the flag on? Though, the flag is off by default and the original UserDefaults settings will still be around.

I think we're good, not an issue! Just wanted to make sure. :)